src/sentry/options/manager.py

```diff
@@ -117,11 +135,27 @@ def lookup_key(self, key):
                 return self.make_key(key, lambda: "", Any, DEFAULT_FLAGS, 0, 0, None)
             raise UnknownOption(key)
 
-    def make_key(self, name, default, type, flags, ttl, grace, grouping_info):
+    def make_key(
+        self,
+        name,
+        default,
+        type,
+        flags,
+        ttl,
+        grace,
+        grouping_info,
+    ):
```

Comment:
Since you're touching this method there's a good opportunity to add type hinting.  E.g. we know it returns key so `-> Key:` could be applied here.  If the other parameters have well known values you could add type hinting there as well.

Reply:
I added types around the manager module. Could not simply add the return type here because this module cannot depend on the module that defines the Key. I'd have to add the typehint as a string.